<a id='497f9091-fb84-47e5-aab4-4a11e9c3451d'></a>

SPECIAL SECTION

<a id='bdd97398-620f-4176-8f37-72152747bcbe'></a>

COMPLETING THE HUMAN GENOME

<a id='9d595fbe-8f52-467f-a68c-3650b1df3220'></a>

RESEARCH ARTICLE

<a id='231d8e10-71de-41b8-bd31-4c766bd32286'></a>

HUMAN GENOMICS

<a id='5bb5eefc-3ac7-40f9-ab71-4a0329a011e2'></a>

The complete sequence of a human genome

<a id='97ab9ee1-9530-40f6-a43e-e379e4863581'></a>

Sergey Nurk¹†, Sergey Koren¹†, Arang Rhie¹†, Mikko Rautiainen¹†, Andrey V. Bzikadze², Alla Mikheenko³,
Mitchell R. Vollger⁴, Nicolas Altemose⁵, Lev Uralsky⁶,⁷, Ariel Gershman⁸, Sergey Aganezov⁹‡,
Savannah J. Hoyt¹⁰, Mark Diekhans¹¹, Glennis A. Logsdon⁴, Michael Alonge⁹, Stylianos E. Antonarakis¹²,
Matthew Borchers¹³, Gerard G. Bouffard¹⁴, Shelise Y. Brooks¹⁴, Gina V. Caldas¹⁵, Nae-Chyun Chen⁹,
Haoyu Cheng¹⁶,¹⁷, Chen-Shan Chin¹⁸, William Chow¹⁹, Leonardo G. de Lima¹³, Philip C. Dishuck⁴,
Richard Durbin¹⁹,²⁰, Tatiana Dvorkina³, Ian T. Fiddes²¹, Giulio Formenti²²,²³, Robert S. Fulton²⁴,
Arkarachai Fungtammasan¹⁸, Erik Garrison¹¹,²⁵, Patrick G. S. Grady¹⁰, Tina A. Graves-Lindsay²⁶,
Ira M. Hall²⁷, Nancy F. Hansen²⁸, Gabrielle A. Hartley¹⁰, Marina Haukness¹¹, Kerstin Howe¹⁹,
Michael W. Hunkapiller²⁹, Chirag Jain¹³⁰, Miten Jain¹¹, Erich D. Jarvis²²,²³, Peter Kerpedjiev³¹,
Melanie Kirsche⁹, Mikhail Kolmogorov³², Jonas Korlach²⁹, Milinn Kremitzki²⁶, Heng Li¹⁶,¹⁷,
Valerie V. Maduro³³, Tobias Marschall³⁴, Ann M. McCartney¹, Jennifer McDaniel³⁵, Danny E. Miller⁴,³⁶,
James C. Mullikin¹⁴,²⁸, Eugene W. Myers³⁷, Nathan D. Olson³⁵, Benedict Paten¹¹, Paul Peluso²⁹,
Pavel A. Pevzner³², David Porubsky⁴, Tamara Potapova¹³, Evgeny I. Rogaev⁶,⁷,³⁸,³⁹, Jeffrey A. Rosenfeld⁴⁰,
Steven L. Salzberg⁹,⁴¹, Valerie A. Schneider⁴², Fritz J. Sedlazeck⁴³, Kishwar Shafin¹¹, Colin J. Shew⁴⁴,
Alaina Shumate⁴¹, Ying Sims¹⁹, Arian F. A. Smit⁴⁵, Daniela C. Soto⁴⁴, Ivan Sović²⁹,⁴⁶, Jessica M. Storer⁴⁵,
Aaron Streets⁵,⁴⁷, Beth A. Sullivan⁴⁸, Françoise Thibaud-Nissen⁴², James Torrance¹⁹, Justin Wagner³⁵,
Brian P. Walenz¹, Aaron Wenger²⁹, Jonathan M. D. Wood¹⁹, Chunlin Xiao⁴², Stephanie M. Yan⁴⁹,
Alice C. Young¹⁴, Samantha Zarate⁹, Urvashi Surti⁵⁰, Rajiv C. McCoy⁴⁹, Megan Y. Dennis⁴⁴,
Ivan A. Alexandrov³⁷,⁵¹, Jennifer L. Gerton¹³,⁵², Rachel J. O'Neill¹⁰, Winston Timp⁸,⁴¹, Justin M. Zook³⁵,
Michael C. Schatz⁹,⁴⁹, Evan E. Eichler⁴,⁵³*, Karen H. Miga¹¹,⁵⁴*, Adam M. Phillippy¹*

<a id='a2781bba-4eb0-452b-9eba-9b1ef4b6d35b'></a>

Since its initial release in 2000, the human reference genome has covered only the euchromatic fraction of the genome, leaving important heterochromatic regions unfinished. Addressing the remaining 8% of the genome, the Telomere-to-Telomere (T2T) Consortium presents a complete 3.055 billion-base pair sequence of a human genome, T2T-CHM13, that includes gapless assemblies for all chromosomes except Y, corrects errors in the prior references, and introduces nearly 200 million base pairs of sequence containing 1956 gene predictions, 99 of which are predicted to be protein coding. The completed regions include all centromeric satellite arrays, recent segmental duplications, and the short arms of all five acrocentric chromosomes, unlocking these complex regions of the genome to variational and functional studies.

<a id='06bd25a9-9a8b-4704-86e5-834a7ecc9313'></a>

The current human reference genome was released by the Genome Reference Consortium (GRC) in 2013 and most recently patched in 2019 (GRCh38.p13) (1). This reference traces its origin to the publicly

<a id='1bae5727-03bc-4e4c-a311-9d9e40a7c664'></a>

funded Human Genome Project (2) and has
been continually improved over the past two
decades. Unlike the competing Celera effort
(3) and most modern sequencing projects
based on "shotgun" sequence assembly (4),

<a id='c95f179c-bbcb-4d37-ad57-26d8f81e1b63'></a>

the GRC assembly was constructed from se-quenced bacterial artificial chromosomes (BACs) that were ordered and oriented along the human genome by means of radiation hy-brid, genetic linkage, and fingerprint maps. However, limitations of BAC cloning led to an underrepresentation of repetitive sequences, and the opportunistic assembly of BACs de-rived from multiple individuals resulted in a mosaic of haplotypes. As a result, several GRC assembly gaps are unsolvable because of in-compatible structural polymorphisms on their flanks, and many other repetitive and poly-morphic regions were left unfinished or in-correctly assembled (5).

<a id='6dd62463-7698-4a8c-a467-4e7db94ceb1a'></a>

The GRCh38 reference assembly contains
151 mega-base pairs (Mbp) of unknown se-
quence distributed throughout the genome,
including pericentromeric and subtelomeric
regions, recent segmental duplications, ampli-
conic gene arrays, and ribosomal DNA (rDNA)
arrays, all of which are necessary for funda-
mental cellular processes (Fig. 1A). Some of the
largest reference gaps include human satellite
(HSat) repeat arrays and the short arms of all
five acrocentric chromosomes, which are repre-
sented in GRCh38 as multimegabase stretches
of unknown bases (Fig. 1, B and C). In addi-
tion to these apparent gaps, other regions of
GRCh38 are artificial or are otherwise in-
correct. For example, the centromeric alpha
satellite arrays are represented as computa-
tionally generated models of alpha satellite
monomers to serve as decoys for resequencing
analyses (6), and sequence assigned to the
short arm of chromosome 21 appears falsely
duplicated and poorly assembled (7). When
compared with other human genomes, GRCh38
also shows a genome-wide deletion bias that
is indicative of incomplete assembly (8). De-
spite finishing efforts from both the Human
Genome Project (9) and GRC (1) that improved
the quality of the reference, there was limited

<a id='45cd81db-3687-4e9e-8b49-42edf47a189b'></a>

1Genome Informatics Section, Computational and Statistical Genomics Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD, USA. 2Graduate Program in Bioinformatics and Systems Biology, University of California, San Diego, La Jolla, CA, USA. 3Center for Algorithmic Biotechnology, Institute of Translational Biomedicine, Saint Petersburg State University, Saint Petersburg, Russia. 4Department of Genome Sciences, University of Washington School of Medicine, Seattle, WA, USA. 5Department of Bioengineering, University of California, Berkeley, Berkeley, CA, USA. 6Sirius University of Science and Technology, Sochi, Russia. 7Vavilov Institute of General Genetics, Moscow, Russia. 8Department of Molecular Biology and Genetics, Johns Hopkins University, Baltimore, MD, USA. 9Department of Computer Science, Johns Hopkins University, Baltimore, MD, USA. 10Institute for Systems Genomics and Department of Molecular and Cell Biology, University of Connecticut, Storrs, CT, USA. 11UC Santa Cruz Genomics Institute, University of California, Santa Cruz, Santa Cruz, CA, USA. 12University of Geneva Medical School, Geneva, Switzerland. 13Stowers Institute for Medical Research, Kansas City, MO, USA. 14NIH Intramural Sequencing Center, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD, USA. 15Department of Molecular and Cell Biology, University of California, Berkeley, Berkeley, CA, USA. 16Department of Data Sciences, Dana-Farber Cancer Institute, Boston, MA, USA. 17Department of Biomedical Informatics, Harvard Medical School, Boston, MA, USA. 18DNAnexus, Mountain View, CA, USA. 19Wellcome Sanger Institute, Cambridge, UK. 20Department of Genetics, University of Cambridge, Cambridge, UK. 21Inscripta, Boulder, CO, USA. 22Laboratory of Neurogenetics of Language and The Vertebrate Genome Lab, The Rockefeller University, New York, NY, USA. 23Howard Hughes Medical Institute, The Rockefeller University, New York, NY, USA. 24Department of Genetics, Washington University School of Medicine, St. Louis, MO, USA. 25University of Tennessee Health Science Center, Memphis, TN, USA. 26McDonnell Genome Institute, Washington University in St. Louis, St. Louis, MO, USA. 27Department of Genetics, Yale University School of Medicine, New Haven, CT, USA. 28Comparative Genomics Analysis Unit, Cancer Genetics and Comparative Genomics Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD, USA. 29Pacific Biosciences, Menlo Park, CA, USA. 30Department of Computational and Data Sciences, Indian Institute of Science, Bangalore KA, India. 31Reservoir Genomics LLC, Oakland, CA, USA. 32Department of Computer Science and Engineering, University of California, San Diego, La Jolla, CA, USA. 33Undiagnosed Diseases Program, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD, USA. 34Heinrich Heine University Düsseldorf, Medical Faculty, Institute for Medical Biometry and Bioinformatics, Düsseldorf, Germany. 35Biosystems and Biomaterials Division, National Institute of Standards and Technology, Gaithersburg, MD, USA. 36Department of Pediatrics, Division of Genetic Medicine, University of Washington and Seattle Children's Hospital, Seattle, WA, USA. 37Max Planck Institute of Molecular Cell Biology and Genetics, Dresden, Germany. 38Department of Psychiatry, University of Massachusetts Medical School, Worcester, MA, USA. 39Faculty of Biology, Lomonosov Moscow State University, Moscow, Russia. 40Cancer Institute of New Jersey, New Brunswick, NJ, USA. 41Department of Biomedical Engineering, Johns Hopkins University, Baltimore, MD, USA. 42National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, MD, USA. 43Human Genome Sequencing Center, Baylor College of Medicine, Houston, TX, USA. 44Genome Center, MIND Institute, Department of Biochemistry and Molecular Medicine, University of California, Davis, CA, USA. 45Institute for Systems Biology, Seattle, WA, USA. 46Digital BioLogic d.o.o., Ivanić-Grad, Croatia. 47Chan Zuckerberg Biohub, San Francisco, CA, USA. 48Department of Molecular Genetics and Microbiology, Duke University School of Medicine, Durham, NC, USA. 49Department of Biology, Johns Hopkins University, Baltimore, MD, USA. 50Department of Pathology, University of Pittsburgh, Pittsburgh, PA, USA. 51Research Center of Biotechnology of the Russian Academy of Sciences, Moscow, Russia. 52Department of Biochemistry and Molecular Biology, University of Kansas Medical School, Kansas City, MO, USA. 53Howard Hughes Medical Institute, University of Washington, Seattle, WA, USA. 54Department of Biomolecular Engineering, University of California, Santa Cruz, Santa Cruz, CA, USA.
*Corresponding author. Email: eee@gs.washington.edu (E.E.E.); khmiga@ucsc.edu (K.H.M.); adam.phillippy@nih.gov (A.M.P.)
†These authors contributed equally to this work. ‡Present address: Oxford Nanopore Technologies Inc., Lexington, MA, USA.

<a id='e5c26ae6-222e-4961-a4b4-8ba3c8da5d63'></a>

Nurk et al., Science 376, 44–53 (2022)

<a id='f239db10-1026-463c-981e-8dcd4991e8fd'></a>

1 April 2022

<a id='ba54c365-2b30-4ff5-90e2-796442cc8f6c'></a>

1 of 10

<a id='eeb3153d-3b74-4edd-b0f3-cc1e8b07378a'></a>

Downloaded from https://www.science.org on April 30, 2025